.devx-cicd/config.yaml

```diff
@@ -0,0 +1,12 @@
+build:
+  - image: 'adoptopenjdk/openjdk8:slim'
+    commands:
+      - './gradlew -x test'
+publishSnapshot:
+  - image: 'adoptopenjdk/openjdk8:slim'
+    commands:
+      - './gradlew -PsymphonyRepoUrl=https://repo.symphony.com/artifactory/libs-snapshot-local -PsymphonyRepoUser=$ARTIFACTORY_USERNAME -PsymphonyRepoPassword=$ARTIFACTORY_PASSWORD publish'
```

Comment:
I think we were trying to avoid leaking the internal artifactory URL in this public Git repo (but this is probably ok)

I wish we could get rid of those property and just `gradle publish` using something similar to the m2/settings.xml file

Reply:
if you're uncomfortable with it, we could add some env variables to devx-cicd project